server/metadata/dictionary.go

```diff
@@ -136,7 +136,7 @@ func (r *reservedSubspace) reload(ctx context.Context, tx transaction.Tx) error
 	}
 
 	var row kv.KeyValue
-	for it.Next(&row) {
+	for it.Next(ctx, &row) {
```

Comment:
There is context passed during iterator creation, we can persist it in the iterator:
```
func (tx *KeyValueTx) Read(ctx context.Context, table []byte, key Key) (Iterator, error) {
	iter, err := tx.ftx.Read(ctx, table, key)
	if err != nil {
		return nil, err
	}
	return &KeyValueIterator{
        ctx: ctx // <- add new field in the iterator struct.
		baseIterator: iter,
	}, nil
}
```
so we don't need to change iterator interface.